sendonly/webrtc-recvonly-h264.c

```diff
@@ -649,7 +650,7 @@ get_string_from_json_object (JsonObject * object)
   return text;
 }
 
-
+#ifdef G_OS_UNIX
 gboolean
 exit_sighandler (gpointer user_data)
```

Comment:
Why is this not `static`?

Reply:
Ah, sorry, didn't see your comments. Just saw that the original reporter confirmed the fix and I hit the button: https://github.com/msys2/MSYS2-packages/issues/2017#issuecomment-647482766